app/views/shelters/edit.html.erb

```diff
@@ -0,0 +1,9 @@
+<h2>Fill out this form to update a shelter!</h2>
+<%= form_tag "/shelters/#{params[:id].to_i}", method: 'patch' do %>
```

Comment:
- method provides the HTTP verb to be used in building the route
- path provided after form_tag will be used to build the route with the verb